apps/docs/src/basics/iconography.md

```diff
@@ -1,3 +1,68 @@
 # Iconography
 
+:::warning Attention
+Check out the whole icon palette in the [resources](/resources/icons) documentation.
+Feel free to browse through all the assets there.
+:::
+
+Our system icons are designed to be simple with a touch of detail. They are characteristic yet easy to understand.
+
+![Image](/assets/icons-overview.png)
+
+## Grid and keyline shapes
+
+The square format with the 1px grid is the foundation for all of our system icons. This ensures the look of the set in terms of thickness, proportion, shape and weight are the same. The grid helps to design a unified set of icons and still be as flexible as possible.
+
+![Image](/assets/icons-grid.png)
+
+## Basic layout
+
+the icons are created on a base grid of 32x32px and then reduced or enlarged to the appropriate size. The grid comes with a padding of 2 px. The padding should only be exceeded if it is necessary to give additional visual weighting or if a certain element is required to give the icon its characteristic or meaning.
+
+![Image](/assets/icons-layout-1.png)
+
+![Image](/assets/icons-layout-2.png)
+
+The key lines shapes ensure a certain consistency in shape and proportion in the overall icon set. In exceptional cases and if the design of the icon does not allow otherwise, the design can go beyond the shape.
```

Comment:
The wording "The key lines shapes" is a little off or don't I understand the sentence? :D

Reply:
This is kept from the already existing guidelines, that were definied earlier (not by myself). It is about the shapes, that are visualizing the the key lines. And those key-line-shapes ensure certain consistency...